k8s/workloads/refractr/refractr-ingress.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   chart:
     repository: https://kubernetes.github.io/ingress-nginx
     name: ingress-nginx
-    version: "3.3.0"
+    version: "3.7.1"
```

Comment:
will this create downtime for refractr?  I thought we split this out to prevent every deploy from creating downtime, and then needed to something (from a medium post?) to get it to be zero downtime.

Reply:
This is for stage i'm testing something atm